src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -2282,7 +2287,10 @@ namespace SurfaceGeometry {
                     if (state.dataHeatBal->Zone(ZoneNum).AllSurfaceFirst == 0) {
                         state.dataHeatBal->Zone(ZoneNum).AllSurfaceFirst = SurfNum;
                     }
-                    if (state.dataSurface->Surface(SurfNum).IsAirBoundarySurf) continue;
+                    if (state.dataSurface->Surface(SurfNum).IsAirBoundarySurf) {
+                        state.dataSurface->Surface(SurfNum).HeatTransSurf = false;
+                        continue;
```

Comment:
Need to set `HeatTransSurf = false` sooner here so that the HT surface lists are built correctly.

Reply:
:+1: